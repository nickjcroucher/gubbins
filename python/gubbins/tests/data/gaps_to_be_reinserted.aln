>sequence_1
AAAAAAAAAAAAAAAAAAAA
>sequence_2
GGGGGGGGGGGGGGGGGGGG
>sequence_3
CCCCCCCCCCCCCCCCCCCC
>sequence_4
TTTTTTTTTTTTTTTTTTTT
>sequence_5
AAAAAAAAAAAAAAAAAAAA
>sequence_6
GGGGGGGGGGGGGGGGGGGG
>sequence_7
AAAAAAAAAAAAAAAAAAAA
>sequence_8
GGGGGGGGGGGGGGGGGGGG
>sequence_9
CCCCCCCCCCCCCCCCCCCC
>sequence_10
TTTTTTTTTTTTTTTTTTTT
>N1
AAAAAAAAAAAAAAAAAAAA
>N2
GGGGGGGGGGGGGGGGGGGG
>N3
CCCCCCCCCCCCCCCCCCCC
>N4
TTTTTTTTTTTTTTTTTTTT
>N5
AAAAAAAAAAAAAAAAAAAA
>N6
GGGGGGGGGGGGGGGGGGGG
>N7
AAAAAAAAAAAAAAAAAAAA
>N8
GGGGGGGGGGGGGGGGGGGG
>N9
CCCCCCCCCCCCCCCCCCCC
>N10
TTTTTTTTTTTTTTTTTTTT