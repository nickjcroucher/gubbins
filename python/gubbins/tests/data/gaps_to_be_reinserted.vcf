##fileformat=VCFv4.1
##FORMAT=<ID=AB,Number=1,Type=String,Description="Alt Base">
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	sequence_1	sequence_2	sequence_3	sequence_4	sequence_5	sequence_6	sequence_7	sequence_8	sequence_9	sequence_10	
1	1	.	A	C,-	.	.	.	AB	A	C	-	-	-	-	-	-	-	A	
1	6	.	A	-	.	.	.	AB	A	-	C	-	-	-	-	-	-	A	
1	8	.	C	-	.	.	.	AB	A	-	C	-	-	-	-	-	-	A	
1	9	.	A	-	.	.	.	AB	A	-	-	C	-	-	-	-	-	A	
1	11	.	A	-,C	.	.	.	AB	A	-	-	C	-	-	-	-	-	A	
1	12	.	A	-,C,N	.	.	.	AB	A	-	-	-	C	-	-	-	-	A	
1	13	.	A	-,N	.	.	.	AB	A	C	-	-	-	-	-	-	-	A	
1	14	.	A	-,C,N	.	.	.	AB	A	-	-	-	C	-	-	-	-	A	
1	18	.	N	C	.	.	.	AB	A	A	A	A	A	A	C	A	A	A	
1	19	.	N	C	.	.	.	AB	A	A	A	A	A	A	C	A	A	A	
1	20	.	N	C	.	.	.	AB	A	A	A	A	A	A	C	A	A	A	
1	26	.	N	C	.	.	.	AB	A	A	A	A	C	A	A	A	A	A	
1	27	.	N	C	.	.	.	AB	A	A	A	A	C	A	A	A	A	A	
1	28	.	N	C	.	.	.	AB	A	A	A	A	C	A	A	A	A	A	
1	30	.	N	C	.	.	.	AB	A	A	C	A	A	A	A	A	A	A	
1	31	.	N	C	.	.	.	AB	A	A	C	A	A	A	A	A	A	A	
1	32	.	A	C	.	.	.	AB	A	A	C	A	A	A	A	A	A	A	
1	33	.	C	A	.	.	.	AB	C	A	A	A	A	A	A	A	A	A