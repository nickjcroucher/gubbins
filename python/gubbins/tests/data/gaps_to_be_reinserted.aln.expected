>N1
AACAAAAACCCCCCCCAA
>N2
GACAGGGGCCCCCCCCGG
>N3
CACACCCCCCCCCCCCCC
>N4
TACATTTTCCCCCCCCTT
>N5
AACAAAAACCCCCCCCAA
>N6
GACAGGGGCCCCCCCCGG
>N7
AACAAAAACCCCCCCCAA
>N8
GACAGGGGCCCCCCCCGG
>N9
CACACCCCCCCCCCCCCC
>N10
TACATTTTCCCCCCCCTT
